packages/rocketchat-lib/server/models/_BaseCache.js

```diff
@@ -744,6 +747,154 @@ class ModelsBaseCache extends EventEmitter {
 		}
 	}
 
+	updateRecord(record, update) {
```

Comment:
I don't think so. We should be updated with the latest mongodb version as we are now.